internal_testsuites/golang/testsuite/startlark_user_passing_test/starlark_user_passing_test.go

```diff
@@ -14,7 +14,7 @@ const (
 	userOverrideServiceName = "user-override"
 
 	starlarkScriptWithUserIdPassed = `
-IMAGE = "hyperledger/besu:latest"
+IMAGE = "hyperledger/besu:24.3"
```

Comment:
This was causing a root user test to fail - it seems in the latest images of besu the default user is `root` so I pinned it to a prior version we were depending on for that test.